compiler/interface.go

```diff
@@ -147,7 +147,7 @@ func (c *compilerContext) getTypeCode(typ types.Type) llvm.Value {
 				c.addError(token.NoPos, fmt.Sprintf("too many levels of pointers for typecode: %s", typstr))
 			}
 			return llvm.ConstGEP(c.ctx.Int8Type(), ptr, []llvm.Value{
-				llvm.ConstInt(llvm.Int32Type(), 1, false),
```

Comment:
@dgryski I believe this was you. Just so you know, the `llvm.*Type()` methods should never be used (see https://github.com/tinygo-org/tinygo/pull/1744 for another instance and https://github.com/tinygo-org/go-llvm/pull/54 where I remove this function).